pkg/ipcache/metadata.go

```diff
@@ -429,7 +433,7 @@ func (ipc *IPCache) InjectLabels(ctx context.Context, modifiedPrefixes []netip.P
 
 	// Recalculate policy first before upserting into the ipcache.
 	if len(idsToAdd) > 0 {
-		ipc.UpdatePolicyMaps(ctx, idsToAdd, idsToDelete)
+		ipc.UpdatePolicyMaps(ctx, idsToAdd, nil)
```

Comment:
How is this related to "ipcache: Only update policy maps for new identities " commit?

Reply:
It’s not directly, just a small error I noticed that causes an unnecessary policy update.

Reply:
Let's pull this out to a separate PR so we can review it separately

Reply:
Sounds good, I'll pull out the `ipcache: Only update policy maps for new identities` commit into a separate PR. 

This particular change highlighted here should be a no-op, as `idsToDelete` is empty at this call site.